src/components/Header/Header.tsx

```diff
@@ -28,9 +28,9 @@ export default function Header() {
 
   return (
     <PageHeader>
-      <PageHeaderTitle title='Custom Repositories' />
-      <Text className={classes.subtext} ouiaId='custom_repositories_description'>
-        View all custom repositories within your organization.
+      <PageHeaderTitle title='Repositories' />
+      <Text className={classes.subtext} ouiaId='repositories_description'>
```

Comment:
Do not change this "ouiaId", these are used in the tests that check our application, changing this will break those tests.